python/tests/templates/test_templates.py

```diff
@@ -20,7 +20,7 @@
 from pydantic import BaseModel, ValidationError
 
 from beeai_framework.utils.errors import PromptTemplateError
```

Comment:
Please move everything that relates to `PromptTemplate` to `beeai_framework/template.py`, everything, including `PromptTemplateError`.